erpnext/patches/v7_0/convert_timelog_to_timesheet.py

```diff
@@ -30,7 +30,12 @@ def execute():
 		time_sheet.flags.ignore_validate = True
 		time_sheet.flags.ignore_links = True
 		time_sheet.save(ignore_permissions=True)
-
+		frappe.db.sql("""update tabCommunication 
```

Comment:
@nabinhait oh forgot about this one ive allready improved so can match the links and do timestamps will update tommorrow